src/features/channel-state.js

```diff
@@ -31,20 +31,22 @@ var resetCountDown = function() {
 var initiateCountDown = function(length) {
     if (bttv.chat.store.chatCountDown) clearInterval(bttv.chat.store.chatCountDown);
 
-    var timer = length;
+    var startTimestamp = Date.now();
+    var endTimestamp = startTimestamp + (length * 1000);
 
     bttv.chat.store.chatCountDown = setInterval(function() {
+        var currentTimestamp = Date.now();
+        var remainingTime = endTimestamp - currentTimestamp;
```

Comment:
You're setting a variable and then using it one time (and never again) in the next line. Just use `Date.now()` in the next line.